templates/sponsorshipslead.html

```diff
@@ -68,7 +68,7 @@ <h3>Make an Impact as Our Sponsorships Lead!</h3>
           LinkedIn, pipeline tools and CRMs (HubSpot), Zoom, social media, Dropbox, Slack, Google 
           Analytics, and Google Workspace tools.</li>
         <li>Are proficient in spoken and written English.</li>
-        <li>Respond to emails and Slack messages promptly.</li>
+        <li>Respond to emails and Slack messages promptly during business hours.</li>
```

Comment:
I accidentally hit enter too early on this commit message, but I don't think it's worth a force push. I'm adding this because, without the caveat to what you really mean here, this was a red flag that almost caused me to not apply. I asked about this in every single interview, because what a lot of people mean when they say this without the caveat is 24/7. Having now been in this role, I'm much more confident that "during business hours" is really what you mean.